src/TLuaInterpreter.cpp

```diff
@@ -4659,6 +4659,14 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+int TLuaInterpreter::getMudletLuaDefaultPath( lua_State * L )
+{
+    QString path = LUA_DEFAULT_PATH "/";
```

Comment:
Ahh, I didn't know about Qt's `QT_NO_CAST_FROM_ASCII`. That's an interesting way to ensure all strings are explicitly marked for translation (or marked as not user-facing). Thanks for explaining. I'll change this to use QStringLiteral.

> Note that some MSVC versions fail to handle QStringLiteral( "stringa" "stringb" )

MSVC doesn't correctly concatenate string literals? :frowning: